src/EditorFeatures/Test/CommentSelection/AbstractToggleCommentTestBase.cs

```diff
@@ -21,28 +21,30 @@
 
 namespace Microsoft.CodeAnalysis.Editor.UnitTests.CommentSelection
 {
-    public abstract class AbstractToggleBlockCommentTestBase
+    public abstract class AbstractToggleCommentTestBase
     {
-        abstract internal ToggleBlockCommentCommandHandler GetToggleBlockCommentCommandHandler(TestWorkspace workspace);
+        abstract internal AbstractCommentSelectionBase<ValueTuple> GetToggleCommentCommandHandler(TestWorkspace workspace);
 
-        protected void ToggleBlockComment(string markup, string expected)
+        protected void ToggleComment(string markup, string expected, bool vbWorkspace = false)
         {
-            ToggleBlockCommentMultiple(markup, new string[] { expected });
+            ToggleCommentMultiple(markup, new string[] { expected }, vbWorkspace);
         }
 
-        protected void ToggleBlockCommentMultiple(string markup, string[] expectedText)
+        protected void ToggleCommentMultiple(string markup, string[] expectedText, bool vbWorkspace = false)
```

Comment:
vbWorkspace is fine, but I think other test usually has like abstract Workspace CreateWorkspace(marker ...) that each language specific test override.